server/aws-lsp-codewhisperer/src/language-server/chat/chatEventParser.ts

```diff
@@ -40,6 +40,12 @@ export class ChatEventParser implements ChatResult {
     static mapReferenceData(reference: Reference): ReferenceTrackerInformation {
         return {
             ...reference,
+            recommendationContentSpan: reference.recommendationContentSpan
+                ? {
+                      start: reference.recommendationContentSpan.start ?? 0,
+                      end: reference.recommendationContentSpan.end ?? 0,
+                  }
+                : undefined,
```

Comment:
Set defaults, as `Reference.recommendationContentSpan` interface was changed, making `start` and `end` fields optional.

This logic needs to be verified or changed before merging

Reply:
@ege0zcan @makenneth could you check this approach and change it if required, since you have more knowledge about expected business logic?

Reply:
I verified it and it is fine to have default values.